NeuralAmpModeler/NeuralAmpModeler.cpp

```diff
@@ -15,44 +16,50 @@
 
 using namespace iplug;
 using namespace igraphics;
+namespace fs = std::filesystem;
```

Comment:
Oh dear...there are _so many_ changes in this due to formatting.

I need this to be formatted correctly before I can review.

Reply:
new commit with fixed formatting